src/swsssdk/interface.py

```diff
@@ -280,7 +280,7 @@ def keys(self, db_name, pattern='*'):
             return keys
 
     @blockable
```

Comment:
>blockable [](http://example.com/codeflow?start=5&length=9)

We are deprecate the python implementation and in favor of swss-common (SWIG powered). Would you like to change in that repo at the same time or only in that repo?

Reply:
Made another PR for that here: https://github.com/Azure/sonic-swss-common/pull/566